python/lsst/ts/electrometer/mock_server.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import logging
+import re
+import threading
+import queue
+
+from serial import Serial
+
+
+class ElectrometerSimulator:
+    def __init__(self, port):
+        self.port = port
+        self.log = logging.Logger(__name__)
+        self.buffer = queue.Queue(maxsize=50000)
+        self.command_calls = {"*IDN?": self.get_identification}
+        self.commands = [
+            re.compile(r"^(?P<cmd>\*IDN\?)$")
+        ]
+
+    def cmd_loop(self):
+        while True:
+            self.log.debug("reading message")
+            rec = self.serial.read_until(b"\r")
+            self.log.debug(f"{rec}")
+            decoded_rec = rec.decode().strip()
+            self.log.debug(f"{decoded_rec}")
+            for command in self.commands:
+                self.log.debug(f"{command}")
+                matched_command = command.match(decoded_rec)
+                self.log.debug(f"{matched_command}")
+                if matched_command:
+                    self.log.debug(f"Matched {decoded_rec}")
+                    command_group = matched_command.group('cmd')
```

Comment:
If this is just the command name then please consider "command_name" instead of "command_group".